hw/dv/tools/xcelium/exclude.tcl

```diff
@@ -0,0 +1,11 @@
+// Copyright lowRISC contributors.
```

Comment:
I think comments in tcl file start with #. CI is complaining about the same thing.

Reply:
oh wierd that xcelium is accepting this.
Thank you